nexus/utils/userValidation.ts

```diff
@@ -0,0 +1,73 @@
+import { integer } from 'aws-sdk/clients/cloudfront'
+import { InvalidInput, UserInputError } from '../errors'
+
+const CheckIfMatchPattern = (email?: string, handle?: string): InvalidInput[] => {
+  const handleRegex = /^[a-zA-Z0-9_-]{3,}$/i
+  const emailRegex = /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i
+
+  const invalidFields: InvalidInput[] = []
+
+  if (email && !email.match(emailRegex))
+    invalidFields.push({
+      name: 'email',
+      message: 'profile.error.emailValidationErrorMessage',
+    })
+
+  if (handle && !handle.match(handleRegex))
+    invalidFields.push({
+      name: 'handle',
+      message: 'profile.error.handleValidationErrorMessage',
+    })
+
+  return invalidFields
+}
+
+const CheckIfUnique = async (
+  db: any,
```

Comment:
I think `PrismaClient` is the type here, e.g. https://github.com/Journaly/journaly/blob/e98c199b6265a6f77760add2eba92c914686ba61/nexus/post.ts#L15